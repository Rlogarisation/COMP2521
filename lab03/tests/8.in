8 3 10 10 -5 3 5 -7 2 11 9 4 6 -3 -6
