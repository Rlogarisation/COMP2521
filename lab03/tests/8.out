Tree:
            8
           / \
          /   \
         /     \
        3      10
       / \     / \
      /   \   9  11
     /     \
    -5      5
   / \     / \
  /   \   4   6
 /     \
-7      2
 \     /
 -6   -3

#nodes = 13,  #leaves = 6
Infix  : -7 -6 -5 -3 2 3 4 5 6 8 9 10 11 
Prefix : 8 3 -5 -7 -6 2 -3 5 4 6 10 9 11 
Postfix: -6 -7 -3 2 -5 4 6 5 3 9 11 10 8 
ByLevel: 8 3 10 -5 5 9 11 -7 2 4 6 -6 -3 
